forte/process_manager.py

```diff
@@ -109,6 +109,17 @@ def initialize_queues(self, pipeline_length: int):
         else:
             raise AttributeError("The process manager is not initialized.")
 
+    def set_current_pipeline(self, pipeline):
```

Comment:
Seems like you are adding a new control mechanism by adding a current_pipeline, could you document what how this is used? This will help future developers since this seems to be a very core function to me.

Reply:
Add in c347766